islandora_bookmark.module

```diff
@@ -197,6 +201,33 @@ function islandora_bookmark_block_view($delta = '') {
         $block['pid'] = $pid;
       }
       break;
+
+    case 'islandora_bookmark_list_info':
+      // Block intended for use on the list view page.
```

Comment:
Probably should refine this a bit, it attempts to fire on /islandora-bookmark which causes some warnings to puke.